.github/workflows/publish.yml

```diff
@@ -18,6 +18,10 @@ jobs:
       with:
         go-version: 1.16
 
+    - name: Set the VERSION
+      run: |
+        echo "VERSION=0.1.1-$(date -u +'%Y%m%d-%H-%M-%S')" >> $GITHUB_ENV
```

Comment:
Wondering if this should instead be in the existing `make publish` stuff.  Similar to how `release.yaml` is doing it.  ```
export VERSION=...
```